ocr_performance/google_tesseract_transcripts.py

```diff
@@ -71,35 +73,40 @@ async def transcribe(page_image_file, ocr, trans_dir):
 
 
 async def transcribe_all(engine, trans_dir):
-    '''
+    """
     Scans through the image files in PHOTOS_DIR and invokes the
     transcribe() function on each
-    '''
+    """
     image_files = [x for x in listdir(PHOTOS_DIR) if x.endswith('.jpg')]
     image_files.sort()
 
     for idx, image_file in enumerate(image_files):
         print(f'Processing image<{idx}:{image_file}>')
         await transcribe(PHOTOS_DIR+image_file, ocr=engine, trans_dir=trans_dir)
 
-if __name__ == '__main__':
+    if __name__ == '__main__':
```

Comment:
for this indent, did you want it to be a part of the transcribe_all() function? If not, just remove it and de-indent lines 89 -101